modules/utility/functions/diff

```diff
@@ -5,13 +5,13 @@
 #   Sorin Ionescu <sorin.ionescu@gmail.com>
 #
 
-function diff {
-  if zstyle -t ':prezto:module:utility:diff' color \
-    && (( $+commands[colordiff] )); then
-      command colordiff "$@"
-  else
-    command diff "$@"
-  fi
-}
+# function diff {
```

Comment:
friendly asked: why committing commented code? or do I oversee something?

Reply:
Good question @blanorama! 
In this case, the `function diff {}` wrapper is unnecessary as you recognize. It was added here for consistency with the convention that we follow in Prezto. Please have a look at the [original PR](https://github.com/sorin-ionescu/prezto/pull/1366) by @belak regarding the motivation behind this.

Reply:
ahh, okay, so in general it’s there for the autoload mechanism to work, which is using grep among others; and in this particular case it would not be needed but for consistency we stick to this…
Do I get it right?

Reply:
Actually, it is there just for consistency (documentation, easy to grep, etc.) only. The line is just a comment, it has no other significance for Zsh runtime :)

Autoloading will work regardless of the line `# function diff {` because the file `diff` exists in a directory (`modules/utility/functions`) listed for Zsh to scans for function (specified via `fpath`).

Reply:
Got it, thanks for clarification. 👍